.github/workflows/release.yaml

```diff
@@ -1,4 +1,4 @@
-name: Dev Prod Deployment
+name: Release new version # we should trigger a different workflow instance per plan, can we make the plan a parameter of the environment?
```

Comment:
Sure I can make this change. Let the user input the plan they wanna release, and release for that plan only